### PR TITLE
corrected turn off and on by index to reflect api

### PR DESCRIPTION
**Pull request by teknix**
According to https://ncd.io/fusion-proxr-advanced-quick-start-guide/ using commands `254, 48` and `254, 47` do not exist.

### CODE REVIEW

File: ncd_industrial_relay.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def send_command(self, command, bytes_back):
 			self.combus.write(command)
 			return self.combus.read(bytes_back)
 		else:
-			self.combus.send(command)
+			self.combus.send(bytearray(command))
```
Comment (kensley) [inline ncd_industrial_relay.py:154]: We can attest that this change worked on our team's side

**Comment by kensley** [comment]
@io-Jacob Any traction on this merge?

**Comment by io-Jacob** [comment]
For some reason I wasn't getting notifications on this repo. Its end of day here, but I'll set up a reminder for tomorrow to review. Thanks for the nudge.

**Comment by io-Jacob** [comment]
So, this is my fault as I thought ProXR and Fusion series boards were more directly compatible, but this command is one of the few differences. On ProXR it is 254, 48, x, x. On Fusion it is 254, 148, x, x.

In an ideal world I would work in some inheritance to where you instantiate a Fusion board object or a ProXR board object etc.

This is a pretty big re-write so for time's sake would you both find it acceptable to add an optional kwarg during instantiation to allow you to pass in the that you're using fusion?

It's not as elegant, but its faster, backward compatible (outside of the issue it fixes) and should translate well into the v2 inherited object (Fusion object, ProXR object).

**Comment by teknix** [comment]
I have a ton more commands I have added at this point including some of the Fusion Reactor stuff I need. I have also asked for more docs upstream. I am relatively new to python and NCD boards so I am not really one to comment on all the differences.

**Pull request merged**